### PR TITLE
[MDB IGNORE] Fix High Capacity APC Inheritance

### PR DESCRIPTION
**Pull request by ORCACommander**
## About The Pull Request

Doing some mapping work downstream and I noticed there were both directional and non directional high caps. Code dived to figure out which was the valid one to use and found the directional high caps were not actually inheriting the upgraded power cells do to bad name structure.  Corrected inheritance structure. Updated Kilo and Icebox to use directional high caps as standard and updated several ruin and gateway missions to standard directional high caps

## Why It's Good For The Game

Areas having the appropriate amount of power is a good thing. Also cleans up var edits

## Changelog

:cl:
fix: fixed High capacity inheritance structure
/:cl:

### CODE REVIEW

File: code/modules/power/apc/apc_mapping.dm [hunk 1]
```diff
@@ -21,6 +21,6 @@
 	cell_type = /obj/item/stock_parts/cell/high
 
 MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/power/apc/auto_name, APC_PIXEL_OFFSET)
-MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/power/apc/five_k, APC_PIXEL_OFFSET)
-MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/power/apc/ten_k, APC_PIXEL_OFFSET)
+MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/power/apc/highcap/five_k, APC_PIXEL_OFFSET)
+MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/power/apc/highcap/ten_k, APC_PIXEL_OFFSET)
```
Comment (ShizCalev) [inline code/modules/power/apc/apc_mapping.dm:25]: we should definitely have a test for this holy fuck

File: _maps/RandomRuins/SpaceRuins/spacehotel.dmm [hunk 2]
```diff
@@ -1802,12 +1796,11 @@
...
-/obj/machinery/power/apc/highcap/five_k{
-	name = "Custodial APC";
-	pixel_y = -25
-	},
+/obj/machinery/power/apc/highcap/five_k/directional/south,
 /obj/structure/cable,
-/obj/structure/sink/directional/west,
+/obj/structure/sink/directional/west{
+	name = "Custodial APC"
+	},
```
Comment (san7890) [inline _maps/RandomRuins/SpaceRuins/spacehotel.dmm:1803]: I don't think this var-edit is correct. You applied the name "Custodial APC" to the sink, not to the APC.

**Review by san7890 — CHANGES_REQUESTED** [review]

File: _maps/RandomRuins/SpaceRuins/spacehotel.dmm (hunk 2 above)
Comment (ORCACommander) [reply _maps/RandomRuins/SpaceRuins/spacehotel.dmm]: <div class="tenor-gif-embed" data-postid="12353147" data-share-method="host" data-aspect-ratio="1.3369" data-width="100%"><a href="https://tenor.com/view/fizzer1k-5597-3689-move-along-nothing-to-seehere-gif-12353147">Fizzer1k 5597 GIF</a>from <a href="https://tenor.com/search/fizzer1k-gifs">Fizzer1k GIFs</a></div> <script type="text/javascript" async src="https://tenor.com/embed.js"></script>

**Comment by ORCACommander** [comment]
anyone got any idea why map diff bot keeps on trying to run?

**Comment by san7890** [comment]
It's broken

**Pull request merged**